tests/search/inconsistent_replicas/disjoint_source_only_documents.rb

```diff
@@ -0,0 +1,96 @@
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+require 'search_test'
+require 'doc_generator'
+
+class DisjointSourceOnlyDocuments < SearchTest
+
+  def make_app(disable_merges:)
+    SearchApp.new.sd(SEARCH_DATA + 'music.sd').
```

Comment:
Consider creating a `test.sd` instead with just the title field needed.

Reply:
Fixed